### DISCUSSION TITLE
EFI

### DISCUSSION THREAD

**Discussion by THOMASVGTR** [THOMASVGTR/DCL-HACKINTOSH-DC483G#2] [General]
EFI for DCL Laptop (DC483G)

<hr /><em>This discussion was created from the release <a href='https://github.com/THOMASVGTR/DCL-HACKINTOSH-DC483G/releases/tag/'>EFI</a>.</em>

**Comment by funnhome** [comment]
![20230608_045403](https://github.com/THOMASVGTR/DCL-HACKINTOSH-DC483G/assets/135925734/37aa674c-b3a9-44e1-acab-ea7738092457)
Hi good day  Thank you for your hard work I ve found your EFI for my hp with sane configuration unfortunately it's stuck at 50% boot with apple logo any help on this
